chessBoard.js

```diff
@@ -0,0 +1,7 @@
+const size = 12;
+let hashString1 = (" #").repeat(size/2);
+let hashString2 = ("# ").repeat(size/2);
+
+for (let i = 0; i < size; i++) {
+    i % 2 === 0 ? console.log(hashString1) : console.log(hashString2);
```

Comment:
i don't believe this works for odd numbers. 

could you add some basic interview-style tests to your files? something like:
```js
const chessBoard = (size) => {
  // code here
};

# Tests
chessBoard(0)
chessBoard(1)
chessBoard(8)
chessBoard(5)
```

for interviews, it's fine to not actually use a testing framework like `jest` or whatever, and instead just call the functions with several different inputs testing edge cases and whatnot and then just doing a visual comparison of outputs. the goal here is more about making sure you're testing your code in some way for correctness and that you're exercising edge cases.